README.md

```diff
@@ -9,19 +9,72 @@ UI Component library for developing [DHIS2](http://www.dhis2.org) applications.
 
 --
 
-### Get It! 
+## Get It!
 
 __Latest stable version for your dhis2 release__
 
-> Use @<dhis2-release-version> to get the latest version applicable to that installation
+> Use @{dhis2-release-version} to get the latest version applicable to that installation
 
-For DHIS2 2.25
+For example, if the DHIS2 release version is 2.28, then:
 ```
-npm install --save d2-ui@25
+yarn add d2-ui@28
 ```
 
-For DHIS2 2.24
+## Contributing
+
+### Development
+
+The git repository is located here: [d2-ui github repo](https://github.com/dhis2/d2-ui)
+
+Submit your change as a pull request. Pull requests need to include:
+1. good quality unit tests
+2. relevant changes/additions to the examples
+
+##### Examples
+The repo includes a webpack dev server that provides a page showing most of the d2-ui components. To run these do:
+```
+yarn start
+```
+
+Some of the examples require a local DHIS2 instance. Most developers have their DHIS2 instance running on http://localhost:8080. If this is the case for you, then the examples should just work.
+
+
+### Publishing to npm
+
+The d2-ui library is published as an npm package: https://www.npmjs.com/package/d2-ui
+
+#### A note about d2-ui semver
+The d2-ui version tracks the DHIS2 version. So, when DHIS2 version 2.28 is released, then d2-ui will be upgraded to v28.0.0. Therefore, all version upgrades prior to the next DHIS2 release are generally patch upgrades, even if the changes are considered breaking.
```

Comment:
I see that we cannot adjust the major version of d2-ui when releasing breaking changes, but we should push minor version bumps for "bigger" stuff which are potentially breaking, and just use patch releases for non-breaking stuff.

We should strive to ensure that d2-ui doesn't accidentally and automatically break for consumers of the lib.

Reply:
I'm actually not completely sure what conventions the previous team members used (@Markionium @nicolayr ).  But I guess we can define the conventions according to what we think makes the most sense? Using minor upgrades to indicate a breaking change could definitely be one of those things.

Reply:
The other thing I am not sure of is when exactly we do the major version, and how all the apps with d2-ui as dependency get the correct version.

Reply:
I think each app needs to update itself with the new version, if they need it.